src/ConnectedMode/UI/Resources/Styles.xaml

```diff
@@ -3,9 +3,31 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
              xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0">
+
+    <SolidColorBrush x:Key="Border" Color="#FFD5DFE5"/>
+
+    <Style TargetType="Window">
+        <Setter Property="Background" Value="{DynamicResource {x:Static vsShell:VsBrushes.WindowKey}}" />
```

Comment:
Add this here? `WindowStartupLocation="CenterOwner"`

Reply:
It can not be added in the style, because it is not a DependencyProperty.
![image](https://github.com/user-attachments/assets/db870d24-1045-4685-9538-b2c8698a3021)

Reply:
It's not owner, it's `WindowStartupLocation`